app/src/main/java/com/github/godspeed010/weblib/fragments/WebViewFragment.kt

```diff
@@ -165,6 +253,21 @@ class WebViewFragment : Fragment() {
         startActivity(Intent.createChooser(shareIntent, "Share This Website!"))
     }
 
+    override fun onStop() {
+        super.onStop()
+
+        overwriteSave()
+    }
+
+    override fun onSaveInstanceState(outState: Bundle) {
+        outState?.run {
```

Comment:
I think an `apply {}` block would be more appropriate than `run {}`

I got the following info from [here](https://kotlinlang.org/docs/scope-functions.html#function-selection)
"
Here is a short guide for choosing scope functions depending on the intended purpose:
- Executing a lambda on non-null objects: `let`
- Introducing an expression as a variable in local scope: `let`
- Object configuration: `apply`
- Object configuration and computing the result: `run`
- Running statements where an expression is required: non-extension `run`
- Additional effects: `also`
- Grouping function calls on an object: `with`

"

Reply:
Will make the adjustment, I think I based this also off the [examples](https://developer.android.com/guide/components/activities/activity-lifecycle#save-simple,-lightweight-ui-state-using-onsaveinstancestate) for instance states, as looking back at it, they also use `run` instead of `apply`.